api/mod/BallotGeneration.js

```diff
@@ -112,17 +109,17 @@ const incrementVotesCasted = (squads, ID, FO) => {
 const getRandomInt = (max) => {
   return Math.floor(Math.random() * Math.floor(max));
 };
+/* Commenting this function expression as it as unused and throwing errors with the linter */
```

Comment:
Code was already commented, but it missed the enclosing brackets, which caused errors. If you see any commented code in my changes, it is because it was already commented--just missing a couple things which was causing errors